fem/quadinterpolator.hpp

```diff
@@ -99,26 +105,36 @@ class QuadratureInterpolator
              Vector &q_val, Vector &q_der, Vector &q_det) const;
 
    /// Interpolate the values of the E-vector @a e_vec at quadrature points.
+   template <QVectorLayout>
+   void Values(const Vector &e_vec, Vector &q_val) const;
    void Values(const Vector &e_vec, Vector &q_val) const;
 
    /** @brief Interpolate the derivatives of the E-vector @a e_vec at quadrature
        points. */
+   template <QVectorLayout>
+   void Derivatives(const Vector &e_vec, Vector &q_der) const;
    void Derivatives(const Vector &e_vec, Vector &q_der) const;
 
    /** @brief Interpolate the derivatives in physical space of the E-vector
        @a e_vec at quadrature points. */
+   template <QVectorLayout>
    void PhysDerivatives(const Vector &e_vec, Vector &q_der) const;
+   void PhysDerivatives(const Vector &e_vec, Vector &q_der) const;
```

Comment:
And here -- add separate documentation to clarify the difference.

Reply:
Moved to the `Compute kernels` section with separate documentation.